profiles/repositories.py

```diff
@@ -68,8 +68,9 @@ def get(self, orcid: str) -> OrcidToken:
         try:
             return self.db.session.query(OrcidToken).filter_by(orcid=orcid).one()
         except NoResultFound as exception:
-            raise OrcidTokenNotFound('ORCID token for the ORCID {} not found'.format(orcid)) \
-                from exception
+            msg = 'ORCID token for ORCID {} not found'.format(orcid)
+            LOGGER.info(msg=msg)
```

Comment:
consider logging some of these events as `error` rather than `info` if they are the result of exceptions (or are correlated with those)